server/views/components/header/header.njk

```diff
@@ -0,0 +1,66 @@
+<div class="header js-header">
+  <div class="header__upper-wrap">
+    <div class="header__upper container">
+      <a href="#" class="header__brand">
+        {# TODO: use svg-sprite logo #}
+        <img class="header__logo" src="https://wellcomecollection.org/sites/all/themes/col_base/images/wellcomecollection-logo.svg" />
+      </a>
+      <form class="header__search js-search">
+        <input class="header__input js-search-input js-show-hide-drawer" type="text" placeholder="Search" />
+        <button class="header__button js-show-hide-trigger">
+          <span class="header__button-icon">
+            {# TODO: use svg-sprite search #}
+          </span>
+          <span class="header__button-text">Search</span>
+        </button>
+      </form>
+    </div>
+  </div>
+  <div class="header__lower container js-show-hide js-header-lower">
+    <a href="#"
+      id="header-burger-trigger"
+      class="header__burger-trigger js-show-hide-trigger"
+      aria-controls="header-nav"
+      aria-label="menu">
+      <span></span>
+      <span></span>
+      <span></span>
+    </a>
+    <nav id="header-nav"
+      class="header__nav js-header-nav"
```

Comment:
I had started with `[data-component]`s, but I came up against a situation where I needed one html element to be responsible for two JS components – without doing something hacky with data-attribute parsing I didn't think this would be possible (e.g. `data-component="thingy-one thingy-two"` is just one value – `thingy-one thingy two` rather than `thingy-one` and `thingy-two`) so reverted to the `js-` prefixed class naming convention.